projects/igniteui-angular/src/lib/grids/summaries/grid-summary.ts

```diff
@@ -59,11 +62,12 @@ export class IgxSummaryOperand {
      * ```
      * @memberof IgxSummaryOperand
      */
-    public operate(data: any[] = [], allData: any[] = [], fieldName?: string): IgxSummaryResult[] {
+    public operate(data: any[] = [], allData: any[] = [], fieldName?: string, locale: string = 'en'): IgxSummaryResult[] {
```

Comment:
Shouldn't the locale follow the [BCP47 tags spec](http://www.rfc-editor.org/rfc/bcp/bcp47.txt)?
In any way the default locale in Angular is `en-US` so maybe set it to that.